internal/gatewayapi/translator_test.go

```diff
@@ -62,7 +62,7 @@ func TestTranslate(t *testing.T) {
 			}
 
 			// Add common test fixtures
-			for i := 1; i <= 3; i++ {
```

Comment:
why not user retry package?

Reply:
this func create service resources on the fly so we dont need to manually add them in the YAML